executor/show_placement.go

```diff
@@ -128,23 +130,116 @@ func (e *ShowExec) fetchShowPlacementLabels(ctx context.Context) error {
 	return nil
 }
 
-func (e *ShowExec) fetchShowPlacement(_ context.Context) error {
-	err := e.fetchAllPlacementPolicies()
+func (e *ShowExec) fetchShowPlacementForTable(_ context.Context) (err error) {
+	dbInfo, ok := e.is.SchemaByName(e.DBName)
```

Comment:
I don't think we need to check this. Just follow other functions like `show create table` or `show table status`.

Reply:
dbInfo is used here to get the original name for the database. It's not appropriate to use the db name user passed by because it is case insensitive and may be different with the one stored in meta